db/seeds.rb

```diff
@@ -1,199 +1,200 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+store = Store.create(name: "Dave's Cars")
+
+car = store.cars.create( make: "Chevy#{i}",
+                       model: "El Camino#{i}",
+                       year: "196#{i}",
+                       daily_price: 100 + i,
+                       weekly_price: 600 + i,
+                       description: "Dave met his wife ##{i} in this car.",
+                       image: File.open('app/assets/images/gnar_possum.jpg'))
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.